app/src/main/res/layout/profil.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Extrage te rog constantele de tip string si pune-le in strings.xml. La fel poti face si pentru constantele de dimensiuni si sa le pui in dimens.xml (e.g. 15sp pt text size)

De asemenea, ai putea sa te joci cu padding-uri si margin-uri prin tot layout-ul ca sa-ti mai distantezi un pic view-urile de marginea ecranului :)